lstchain/io/lstcontainers.py

```diff
@@ -51,23 +56,34 @@ class DL1ParametersContainer(Container):
     disp_miss = Field(None, 'disp_miss [m]', unit=u.m)
     src_x = Field(None, 'source x coordinate in camera frame', unit=u.m)
     src_y = Field(None, 'source y coordinate in camera frame', unit=u.m)
-    time_gradient = Field(None, 'Time gradient in the camera')
-    intercept = Field(None, 'Intercept')
-    leakage_intensity_width_1 = Field(None, 'Fraction of intensity in outermost pixels')
-    leakage_intensity_width_2 = Field(None, 'Fraction of intensity in two outermost rings of pixels')
-    leakage_pixels_width_1 = Field(None, 'Fraction of signal pixels that are border pixels')
-    leakage_pixels_width_2 = Field(None, 'Fraction of signal pixels that are in the two outermost rings of pixels')
-    n_pixels = Field(None, 'Number of pixels after cleaning')
-    concentration_cog = Field(None, 'Fraction of intensity in three pixels closest to the cog')
-    concentration_core = Field(None, 'Fraction of intensity inside hillas ellipse')
-    concentration_pixel = Field(None, 'Fraction of intensity in brightest pixel')
-    n_islands = Field(None, 'Number of Islands')
-    alt_tel = Field(None, 'Telescope altitude pointing', unit=u.rad)
-    az_tel = Field(None, 'Telescope azimuth pointing', unit=u.rad)
-
-    obs_id = Field(None, 'Observation ID')
-    event_id = Field(None, 'Event ID')
-    calibration_id = Field(None, 'ID of the employed calibration event')
+    time_gradient = Field(np.nan, 'Time gradient in the camera')
```

Comment:
time gradient has unit `np.nan / u.m`

Reply:
Are we sure this won't break anything down the line? 
I think this is an "Acceptable Deferred Defect". Will be solved when we move to standard DL1.